README.md

```diff
@@ -5,4 +5,7 @@ You can select a size (mass) for each body from three options (dropdown), and an
 To calculate the trajectories, the known closed-form solutions, which can be found in any classical mechanics textbook (such as Landau & Lifshitz), were used. 
 
 ## Install Dependencies
-Install tkinter.
+Run `python pip install tk` to download tkinter to be able to run the app.
```

Comment:
Lol does this actually work? I've always had to do python -m pip or just pip. Cool.

This is perfectly fine but I'd recommend run commands go on there own line like this

Install Tkinter
```
python pip install tk
```